### DISCUSSION TITLE
Invitation for the second committer of api-testing project

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [LinuxSuRen/api-testing#479] [Announcements]
I truly wish @yuluo-yx could be the committer of [api-testing](https://github.com/LinuxSuRen/api-testing) project. He made huge contributions to this project. For me, he always has the passion to make things better. I deeply believe he could be a good maintainer of this project.

Please let's review his contributions:

* [23 PRs](https://github.com/LinuxSuRen/api-testing/pulls?q=is%3Apr+author%3Ayuluo-yx+is%3Aclosed+label%3Alegend)
* [4 Open and 10 Closed Issues](https://github.com/LinuxSuRen/api-testing/issues?q=is%3Aissue+author%3Ayuluo-yx+label%3Alegend)

And he is still trying to [refactor the whole UI](https://github.com/LinuxSuRen/api-testing/pull/466) and [create an official website](https://github.com/LinuxSuRen/api-testing/pull/469).

So thank you for your big effort! I will give you permission to commit once you accept it. And I hope more people can join us in any way.

**Comment by yuluo-yx** [comment]
tks for the invite, I'm honored to be a project committer. 🫡

**Reply by LinuxSuRen** [reply]
![image](https://github.com/LinuxSuRen/api-testing/assets/1450685/2120b6ef-d344-45aa-99ad-073d2ae94288)

**Comment by LinuxSuRen** [comment]
![image](https://github.com/LinuxSuRen/api-testing/assets/1450685/3d17086c-ffea-4dee-b51b-4f63e0a31358)
